### PR TITLE
Add optional cleanup step after failover to HA replica

### PR DESCRIPTION
**Pull request by mwiesen**
### Why:

With the current documentation a failover will be completed and access to the GitHub Enterprise Server will be restored, however a configuration cleanup will not be performed.

This means that Spokes, DPages, and Storage will maintain references to the old (potentially inaccessible) servers yet will be unable to replicate any data to them. This can and will eventually lead to clone/push failures and full repository outages should those servers not be re-added as replicas.

Solves https://github.com/github/docs-content/issues/2564

### What's being changed:

Our current documentation outlines the high-level steps to perform a failover, however more detail is needed clarifying follow-up actions in the event that that the primary server and/or other replicas will not be re-added to the GitHub Enterprise Server configuration.

I added a new (optional) step following the current step 7 noting the UUIDs of any servers which will not be re-added to the environment should be cleaned up by running `ghe-repl-teardown -u <uuid_here>`.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

@maclarel 
@myarb

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR @mwiesen! I'll let the @github/docs-content-core team know it's ready for review 💝

File: content/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance.md
Comment (felicitymay) [inline content/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance.md:41]: Thanks for updating this topic 😄 

After looking at the [topic on staging](https://docs-252--mwiesen-failover-cle.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance), I can see that `<uuid_here>` is treated as an HTML tag and disappears.

I think we can use the form used in other Enterprise admin topics, like: [Creating the GitHub Enterprise Server virtual machine](https://docs-252--mwiesen-failover-cle.herokuapp.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-azure#creating-the-github-enterprise-server-virtual-machine).

```suggestion
    - On the new primary, remove the UUIDs using `ghe-repl-teardown`. Please replace *`UUID`* with a UUID you retrieved in the previous step.
      ```shell
      $ ghe-repl-teardown -u <em>UUNID</em>
      ```
```

**Review by felicitymay — COMMENTED** [review]
One suggestion to ensure that the example command appears in full, otherwise the change looks great 💖

**Review by felicitymay — CHANGES_REQUESTED** [review]
This is ready to merge once the formatting of the example command is fixed 😄

**Comment by felicitymay** [comment]
Hi  @mwiesen - I'm not sure if you saw my comment on this pull request. From your really clear description of the problem this change addresses, I'm keen to resolve the problem so that we can merge this PR and your improvement to the docs will be live. 

If don't hear from you before Monday, October 26, I'll commit my fix and merge this change.

**Review by mwiesen — COMMENTED** [review]
Thanks for spotting that @felicitymay !

**Review by felicitymay — APPROVED** [review]
All looks correct on staging now: https://docs-252--mwiesen-failover-cle.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance

**Comment by felicitymay** [comment]
Updated from `main`. Once the tests complete, this should be ready to merge.

File: content/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance.md
Comment (taz) [inline content/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance.md:40]: `s/UUNID/UUID/` small typo

Comment (taz) [reply content/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance.md:40]: Hmm not sure why this only just popped up in my notifications given it was merged a month ago. I'll open a new PR to fix the typo.

**Pull request merged**
